c++/src/capnp/schema.c++

```diff
@@ -265,6 +265,18 @@ Schema::BrandArgumentList Schema::getBrandArgumentsAtScope(uint64_t scopeId) con
   return BrandArgumentList(scopeId, raw->isUnbound());
 }
 
+kj::Array<uint64_t> Schema::listBrandArgumentScopes() const {
+  KJ_REQUIRE(getProto().getIsGeneric(), "Not a generic type.", getProto().getDisplayName());
```

Comment:
Maybe this should just return an empty array (`return nullptr`)?

Reply:
I was simply following the existing API laid out by getBrandArgumentsAtScope. Returning an empty array makes sense .